docs/Audit/AuditDiffer.md

```diff
@@ -0,0 +1,57 @@
+# AuditDiffer
+
+The `AuditDiffer` is an object differ, specifically built for generating diffs that can go into an audit log.
+
+In list form it...
+
+- will find changes made to `public` fields and properties
+- allows you to exclude certain fields or properties using the `AuditAttribute`
+- allows you to "mask" certain fields or properties using the `AuditAttribute` to set an [IAuditValueMasker](../ValueMasking)
+
+It is built on top of the context agnostic [RapidCore.Diffing.StateChangeFinder](../../Diffing/StateChangeFinder).
+
+Note that it uses `IRapidContainerAdapter` to create `IAuditValueMasker` instances.
```

Comment:
Do we have documentation somewhere on the `IRapidContainerAdapter` and how people should set it up?

If not I suggest we add an TODO dummy page and record an issue to get those docs in

Reply:
No docs yet. It is an extremely simple interface though.